src/02-data-types.js

```diff
@@ -22,7 +24,9 @@ function getLengthOfString(str) {}
  *  convertToNumber("111");
  *  //> 111
  */
-function convertToNumber(val) {}
+function convertToNumber(val) {
+  return +val
+}
```

Comment:
`Number(val)` would be clearer, but others prefer `parseInt` or `parseFloat`. `+` works too, but clarity is king!